po/pt_BR/system-monitor.po

```diff
@@ -303,7 +303,7 @@ msgstr "Estilo de uso"
 
 #: prefs.js:436
 msgid "pie"
-msgstr "setores"
+msgstr "pizza"
 
```

Comment:
Are you sure about this? "Pie" in this context refers to a circular graph. Not the food.

Reply:
Hi! Yes, here in Brazil this type of graph is called "Gráfico de pizza"